### DISCUSSION TITLE
Plan for Polyhedral cell type support for unstructured volumes?

### DISCUSSION THREAD

**Discussion by anskwagh** [RenderKit/ospray#585] [Q&A]
Hello,

**Is there any future plan for polyhedral cell type support for unstructured volume?** It would surely help a lot and reduce multiple challenges from our side.

**Comment by johguenther** [comment]
Support would be via Open VKL. As stated in https://github.com/openvkl/openvkl/issues/16, no concrete plans so far.

**Reply by anskwagh** [reply]
ok. Thanks!
